src/Entity/Mail/Mail.php

```diff
@@ -12,39 +13,46 @@
 class Mail
 {
     /**
+     * @var string
      * @ORM\Id()
      * @ORM\GeneratedValue(strategy="UUID")
      * @ORM\Column(type="guid")
      */
     private $id;
 
     /**
+     * @var LocalAccount | null
      * @ORM\ManyToOne(targetEntity="App\Entity\Security\LocalAccount")
      * @ORM\JoinColumn(name="person_id", referencedColumnName="id")
      */
     private $person;
 
     /**
+     * @var Collection<int,Recipient>
```

Comment:
I'm not sure of this, but I believe this value could be hydrated with type `Collection<int,Recipient>|Recipient[]` (these sort of types occur in more places, also 272)

Reply:
could not find any others in this PR, I've made a not for the remaining one's